heartbeat/ocf.py

```diff
@@ -450,16 +450,16 @@ def value_for_parameter(param):
 		sys.exit(rc)
 	sys.exit(OCF_ERR_UNIMPLEMENTED)
 
-
```

Comment:
You can add back the newline here. 2 lines for separate __main__ from the rest of the code makes sense, we just dont need to add anymore lines.